fineract-provider/src/main/resources/sql/migrations/core_db/V316__address_module_tables_metadat.sql

```diff
@@ -80,12 +80,12 @@ AUTO_INCREMENT=1
 -- field configuration
 CREATE TABLE `m_field_configuration` (
 	`id` BIGINT NOT NULL AUTO_INCREMENT,
-	`entity` VARCHAR(100) NOT NULL,
-	`subentity` VARCHAR(100) NOT NULL,
-	`field` VARCHAR(100) NOT NULL,
+	`entity` varchar(100) NOT NULL,
```

Comment:
there are A LOT of (uppercase) `VARCHAR` in the DB scripts - why do we have to change them for EclipseLink? Better avoid touching what you can if you don't have to, IMHO. If absolutely required, raise incremental small PRs. In this particular, I doubt EclipseLink really needs this.